tests/ethpm/integration/test_ipfs_integration.py

```diff
@@ -26,3 +27,24 @@ def test_local_ipfs_backend_integration_round_trip(monkeypatch):
         local_manifest = f.read()
     ipfs_manifest = backend.fetch_uri_contents(asset_data["Hash"])
     assert ipfs_manifest == local_manifest
+
+
+@pytest.mark.parametrize("backend", (LocalIPFSBackend(), InfuraIPFSBackend()))
```

Comment:
While parametrize does a decent job here, using a parametrized fixture would allow you to lazily only instantiate the backend you need and to have better isolation in the case any backend is stateful.

```python
@pytest.fixture(params=[LocalIPFSBackend, InfuraIPFSBackend])
def backend(request):
    return request.param()
```